bl/policy/policy_factory.go

```diff
@@ -112,14 +154,22 @@ func getDefaultRuleByIdentifier(defaultRules []*internal_policy.DefaultRuleDefin
 	return nil
 }
 
-func getCustomRuleByIdentifier(customRules []*cliClient.CustomRule, identifier string) *cliClient.CustomRule {
+func getCustomRuleByIdentifier(customRules []*cliClient.CustomRule, identifier string) (*cliClient.CustomRule, error) {
+	var chosenPolicy *cliClient.CustomRule
 	for _, customRule := range customRules {
 		if identifier == customRule.Identifier {
-			return customRule
+			if chosenPolicy != nil {
+				// If we are here that means we have found another
+				// identifier with the same name. So, throw error
+
+				err := fmt.Errorf("invalid policy file: identifiers are not unique")
```

Comment:
every function should do one thing, this function has 2 responsibilities:
1. get the customRule
2. validate that every customRule has a unique identifier 

On another note, the validation for rules identifiers uniqueness should be across all default rules **and** custom rules